db/schema.rb

```diff
@@ -754,15 +756,16 @@
     t.datetime "created_at", precision: nil, null: false
     t.datetime "updated_at", precision: nil, null: false
     t.boolean "read", default: false
-    t.index ["author_id"], name: "index_warnings_on_author_id"
```

Comment:
I'm not entirely sure why this changed? I didn't notice it when I was creating the migration, and I'm not sure what this does

Reply:
Forgot to mention here, but on discord I was told that this is just because my version of the database is older and uses a previous name for the index, and it doesn't actually affect how it functions.